### DISCUSSION TITLE
How to setup with my VPS

### DISCUSSION THREAD

**Discussion by wdsmaduranga** [bluenviron/mediamtx#1063] [Questions]
i get vps for this one give me guide for use this one

**Comment by aler9** [comment]
Hello, installation instructions are the same for PC and VPS: https://github.com/aler9/rtsp-simple-server#installation

Regarding VPS, if i were you i'd choose a machine with Ubuntu 22.04 and Docker installed.

**Reply by wdsmaduranga** [reply]
dear sir , please give me command on my vps i tried so many times thank you

[a comment by an automated account is not shown]
